docs/enable_disable.md

```diff
@@ -0,0 +1,42 @@
+# enable/disable repo
+
+You can enable or disable the parsing of any repo defined in `.sgit.yml` file by adding the flag `enable: true/false` to your repo configuration block.
+
+If this flag is not defined or set in your configuration, it will default to `true` and a repo will be parsed by default.
+
+
+## Enable a repo explicitly
+
+```yaml
+repos:
+  pykwalify:
+    enable: true
+    clone-url: git@github.com:Grokzen/pykwalify.git
+    revision:
+      branch: master
+```
+
+
+## Disable repo explicitly
+
+You can disable repo with the following flag
+
+```yaml
+repos:
+  pykwalify:
+    enable: false
```

Comment:
Would it make sense to name this key `enabled` instead of `enable`. This is a question of preference so I am not saying either is better than the other, you decide.

Reply:
I think that ansible is using `enabled` so i can go for that actually.